lib/library/SystemLibraryPlugin.js

```diff
@@ -116,6 +116,13 @@ class SystemLibraryPlugin extends AbstractLibraryPlugin {
 									Template.asString([
 										"function(module) {",
 										Template.indent(`${external} = module;`),
+										Template.indent([
+											"if (!Object.hasOwnProperty.call(module, '__esModule')) {",
+											Template.indent(
+												`Object.defineProperty(${external}, "__esModule", {value: true});`
```

Comment:
Is it legal to modify the namespace objects from System.js? Should we better create a new object with getters to the original object?

Reply:
You are right, it would be better not to modify the systemjs namespace objects. I have updated this with your suggestion to use a separate object + getters.